src/base/utils.js

```diff
@@ -85,6 +85,14 @@ export var Fullscreen = {
     } else if(document.msExitFullscreen) {
       document.msExitFullscreen()
     }
+  },
+  canGoFullscreen: function() {
+    return !(
+      document.fullscreenEnabled === false ||
+      document.webkitFullscreenEnabled === false ||
+      document.mozFullScreenEnabled === false ||
+      document.msFullscreenEnabled === false
+    )
```

Comment:
I was creating an issue so we can discuss that, and found #780. Since this PR solves the issue partially, I'll leave this referenced and create another one so we can approach the full viewport request.